packages/contracts/test/diamond/facets/DirectGovernanceFarmerFacet.t.sol

```diff
@@ -257,27 +348,28 @@ contract DirectGovernanceFarmerTest is LocalTestHelper {
         vm.stopPrank();
 
         // mint 100 Dollars to user
-        vm.prank(admin);
-        dollar.mint(userAddress, 100e18);
+        vm.startPrank(admin);
+        dollar.mint(userAddress, 99e18);
```

Comment:
Why the comment says mint 100 Dollars and 99 Dollars are minted?

Reply:
it is originally done that way, reminder keeping as much original as possible

Reply:
I see originally minted amounts were 100 Dollars, 99 DAI, 98 USDC and 97 USDT and now 99 Dollars, 99 DAI, 98 USDC and 97 USDT. could you please double check if I see this right?

Reply:
Ok once we check and confirm this one I will approve from my side.

Reply:
both values shall be ok, the idea is to meet the assertion https://github.com/ubiquity/ubiquity-dollar/pull/841/files#diff-6140f6576af81654e444ba3b5bddd1fc61cfaf0a5670769f670bd2fb4ecb3527L331 that the user leaves no token in its wallet, because everything its deposited into the staking contract, can you iterate it and  make the sense?

Reply:
i'm recycling a lot of bad commenting of new PR

Reply:
@molecula451 all right, your explanations make sense. Since you are recycling bad commenting on new PR I will approve this one.